woocommerce/woocommerce-android
# [Woo POS] Adapt the UI to be usable on smaller devices

Closes: #11937


### Description

Adapts the UI to be more usable on smaller screens

### Steps to reproduce


### Testing information

Test the changes on different screen size:
* Notice that it looks better on the smallest supported screens
* Make sure no changes are done on the big tablets

### Images/gif


<img width="817" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/9eb2ebc5-7d98-4f51-a1ac-02b3f975b485">
<img width="816" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/69ca66a8-3d36-410c-9270-4de84a9c2d85">
<img width="812" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/c34c063a-7637-4bc8-9113-61b0e051abfd">
